web/themes/custom/pineapple/templates/content/node--author--full.html.twig

```diff
@@ -0,0 +1,104 @@
+ *     teaser listings.
+ *   - node--unpublished: Appears on unpublished nodes visible only to site
+ *     admins.
+ * - title_attributes: Same as attributes, except applied to the main title
+ *   tag that appears in the template.
+ * - content_attributes: Same as attributes, except applied to the main
+ *   content tag that appears in the template.
+ * - author_attributes: Same as attributes, except applied to the author of
+ *   the node tag that appears in the template.
+ * - title_prefix: Additional output populated by modules, intended to be
+ *   displayed in front of the main title tag that appears in the template.
+ * - title_suffix: Additional output populated by modules, intended to be
+ *   displayed after the main title tag that appears in the template.
+ * - view_mode: View mode; for example, "teaser" or "full".
+ * - teaser: Flag for the teaser state. Will be true if view_mode is 'teaser'.
+ * - page: Flag for the full page state. Will be true if view_mode is 'full'.
+ * - readmore: Flag for more state. Will be true if the teaser content of the
+ *   node cannot hold the main body content.
+ * - logged_in: Flag for authenticated user status. Will be true when the
+ *   current user is a logged-in member.
+ * - is_admin: Flag for admin user status. Will be true when the current user
+ *   is an administrator.
+ *
+ * @see template_preprocess_node()
+ *
+ * @todo Remove the id attribute (or make it a class), because if that gets
+ *   rendered twice on a page this is invalid CSS for example: two lists
+ *   in different view modes.
+ */
+#}
+{%
+  set classes = [
+    'node',
+    'node--type-' ~ node.bundle|clean_class,
+    node.isPromoted() ? 'node--promoted',
+    node.isSticky() ? 'node--sticky',
+    not node.isPublished() ? 'node--unpublished',
+    view_mode ? 'node--view-mode-' ~ view_mode|clean_class,
+  ]
+%}
+{{ attach_library('classy/node') }}
+<article{{ attributes.addClass(classes) }}>
+  <div{{ content_attributes.addClass('author-body') }}>
+    {% set cover_image_url = bamboo_render_image_style(content.field_a_cover_image|field_target_entity.field_media_image.entity.uri.value, 'cover_image') %}
+    <div class="cover-image breakout" style="background-image: url({{ cover_image_url }})"></div>
+    <div class="author-container breakout">
+      <div class="author-info-container container">
+        <div class="profile-image">
+          {% set profile_image_url = bamboo_render_image_style(content.field_a_image|field_target_entity.field_media_image.entity.uri.value, 'thumbnail_115_x_115') %}
+          {% set profile_image_alt = content.field_a_image|field_target_entity.field_media_image.alt %}
+          <img src="{{ profile_image_url }}" alt="{{ profile_image_alt }}">
+        </div>
+        <div class="author-text">
+          <h2 class="author-name">{{ node.getTitle() }}</h2>
+          <p class="author-biography">{{ content.field_a_biography }}</p>
```

Comment:
Since you're only doing content.field_a_biography the bio is being rendered with divs around and since you can't have divs in a p tag you've got empty p tags and then the div with the content below

Reply:
Done! This was my mistake.